internal/rpc/convert_test.go

```diff
@@ -0,0 +1,309 @@
+package rpc
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aquasecurity/trivy/pkg/log"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	ptypes "github.com/aquasecurity/go-dep-parser/pkg/types"
+	dbTypes "github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy/pkg/types"
+	"github.com/aquasecurity/trivy/rpc/detector"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	log.InitLogger(false, false)
+	code := m.Run()
+	os.Exit(code)
+}
```

Comment:
For my own learning: What does this do?

Reply:
We can do setup or teardown before or after testing by using `TestMain`
https://golang.org/pkg/testing/#hdr-Main

Reply:
I see. So if we do this does it mean that we can ensure a logger is created and initialized for all the tests in this file?

Reply:
Looks like it indeed is the case. The more you know (TM) 😄